src/Fixer/AttributeNotation/OrderedAttributesFixer.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\AttributeNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\NamespaceAnalysis;
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\NamespaceUseAnalysis;
+use PhpCsFixer\Tokenizer\Analyzer\NamespacesAnalyzer;
+use PhpCsFixer\Tokenizer\Analyzer\NamespaceUsesAnalyzer;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use Symfony\Component\OptionsResolver\Options;
+
+/**
+ * @author HypeMC <hypemc@gmail.com>
+ */
+final class OrderedAttributesFixer extends AbstractFixer implements ConfigurableFixerInterface
```

Comment:
For discussion, not a strict blocker: wouldn't it make more sense to name it `attribute[s]_order`, which would fit better the [Attribute Notation](https://cs.symfony.com/doc/rules/index.html#attribute-notation) category and allow placing this fixer next to [`attribute_empty_parentheses`](https://cs.symfony.com/doc/rules/attribute_notation/attribute_empty_parentheses.html) and other attribute-related fixers? The problem is we currently have quasi-inconsistent naming for it: several `ordered_*` fixers and some ordering fixers related to phpDoc with `phpdoc_*order` so currently proposed name probably fits the convention well, but I am not sure if we should continue it..

Fixers' aliases could be helpful here, also [FQCN as fixers' identifier](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7064) would reduce the time needed to refactor configuration when fixers' get deprecated or renamed.

CC: @keradus

Reply:
I'm fine either way, so please let me know when you decide.

Reply:
@Wirone I do not see the immediate value of aligning locally for this single fixer , to be honest. Please conclude on the path you see as more valuable.

If we want to focus on this, we should overview all fixers and decide on consistent naming rules, then we would know which way to go for this (and any other) fixer.

Reply:
OK, so let's stick with the current name because it follows current convention, and I'll try to propose fixers' mass renaming when I find time for it.